modules/nw-routeadvertisements-example.adoc

```diff
@@ -0,0 +1,254 @@
+// Module included in the following assemblies:
+//
+// * networking/route_advertisements/about-bgp-routing.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-bgp-examples_{context}"]
+= Example advertising pod IP addresses with BGP
+
+The following examples describe several configurations for advertising pod IP addresses with Border Gateway Protocol (BGP). The external network border router has the `172.18.0.5` IP address.
+
+Each example relies upon the following `FRRConfiguration` object:
```

Comment:
I woudl move this to the `== Advertising pod IPs from a cluster user-defined network over BGP` section as specific to that example.